app/src/androidTest/java/com/hedvig/app/feature/loggedin/TerminatedTest.kt

```diff
@@ -45,7 +45,7 @@ class TerminatedTest : TestCase() {
 
     @Test
     fun shouldOpenWithHomeTabWhenUserIsNotTerminated() = run {
-        every { loginStatusService.isLoggedIn }.returns(true)
+        every { loginStatusService.isLoggedIn }.returns(false)
```

Comment:
This doesn't seem to be in line with what the test says 😮

Reply:
Not really sure what this is testing but it was actually returning isLoggedIn == false before refactoring 🤔

Reply:
It's testing that terminated members get to see the logged in-screen with the terminated-state. Is this no longer the case after the refactor?